sample-extension-plugin/src/main/java/org/opensearch/jobscheduler/sampleextension/SampleJobRunner.java

```diff
@@ -32,7 +32,7 @@
  * The job runner should be a singleton class if it uses OpenSearch client or other objects passed
  * from OpenSearch. Because when registering the job runner to JobScheduler plugin, OpenSearch has
  * not invoke plugins' createComponents() method. That is saying the plugin is not completely initalized,
- * and the OpenSearch {@link org.opensearch.client.Client}, {@link ClusterService} and other objects
+ * and the OpenSearch {@link Client}, {@link ClusterService} and other objects
```

Comment:
Typo.

Reply:
`and the OpenSearch {@link org.opensearch.client.Client}` should be changed to `and the OpenSearch {@link org.opensearch.transport.client.Client}`?

Reply:
Since this `org.opensearch.transport.client.Client` is already part of the import `Client` refers to the same package Peter. Thanks